docs/development/methodology/trello.md

```diff
@@ -39,7 +39,7 @@ cards is blue.
 
 ### Use of Trello in Weekly Meetings
 
-Trello is the main way in which we can communicate with the client
+Trello is the main way in which we communicate to the client
```

Comment:
I don't, I think the rest of the sentence refers to us communicating something to the client, not with them which would involve a two-way communication